tasks/delete_queue_test.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import io
+
+import delete_queue
+
+TEST_PROJECT_ID = os.getenv('GOOGLE_CLOUD_PROJECT')
+TEST_LOCATION = os.getenv('TEST_QUEUE_LOCATION', 'us-central1')
+TEST_QUEUE_NAME = os.getenv('TEST_QUEUE_NAME', 'my-queue')
+
+
+def test_delete_queue():
```

Comment:
This can be shortened similar to https://github.com/GoogleCloudPlatform/python-docs-samples/blob/6a2e183ac4efd8f7c1f68c155ccec20cc36a157f/automl/cloud-client/list_datasets_test.py#L26